proxy/src/main.rs

```diff
@@ -190,13 +204,13 @@ fn cli() -> clap::Command {
             Arg::new("mgmt")
                 .short('m')
                 .long("mgmt")
-                .help("listen for management callback connection on ip:port")
-                .default_value("127.0.0.1:7000"),
+                .help("listen for management callback connection on ip:port (disabled by default)")
+                .default_value(""),
```

Comment:
I don't believe `.default_value("")` is what we need here.

* Initially, this value should remain unchanged, allowing us to have two active mgmt endpoints at a time: a legacy one (pq) and a new one (http).
* After adapting the console to use the http endpoint, we should drop the legacy endpoint completely, without changing any of its settings to misleading blank string literals or whatever.

Reply:
Works for me. I figured it would be useful for testing to be able to disable the mgmt interface, even before we remove it completely. But I guess you can also just set it to a different port, to "move it out of sight".